include/zephyr/mgmt/ec_host_cmd/ec_host_cmd.h

```diff
@@ -22,9 +22,11 @@ struct ec_host_cmd {
 	struct ec_host_cmd_rx_ctx rx_ctx;
 	struct ec_host_cmd_tx_buf tx;
 	struct ec_host_cmd_backend *backend;
+#ifdef CONFIG_EC_HOST_CMD_DEDICATED_THREAD
 	struct k_thread *thread;
 	k_thread_stack_t *stack;
 	k_tid_t thread_id;
```

Comment:
Do you need this? Does not seem used now, would be needed to do stuff like stopping the thread but if in doubt I'd lay on the side of dropping it and saving 4 bytes.

Reply:
I needed this for `task_id_to_thread_id` function in the EC repo. But it is nor required anymore with reusing the main thread.